sdk/extensions/tests-install-azurite.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
```

Comment:
@weshaggard, @hallipr:   I'd appreciate your thoughts here.   Not sure if we have a standard way to bootstrap Azureite.

Reply:
@weshaggard, if this is going to be a shared template yml file used by multiple projects, should it live in somewhere under the eng folder? To me it doesn't seem like the correct structure for us to reference a yml file located in a different projects folder as it's not obvious that it's a shared template.

Reply:
I'm going to resolve this now as it should be functioning, but I do still think the tests-install-azurite.yml should be moved under the eng folder for the reasons I states above.